rules/RFC/Project_Cypress_Development_Rules.md

```diff
@@ -0,0 +1,106 @@
+- Use kebab-case for file and folder names: Use lowercase letters separated by hyphens (e.g., user-service.ts, order-processor.ts).
+- Avoid abbreviations: Use full words to avoid confusion (e.g., authentication instead of auth).
+
+## Test Repository Structure
+- Test specs, page objects, scenarios and commands should be logically separated into subfolders based on the business feature they belong to
+- Actions that involve several pages (e.g. click button on one page and then go to another page to see and adjust the result of that click) should be created as Scenarios in `cypress/support/scenarios`
+- Actions within one application page (e.g. clicking a button on that page or editing a filed) should be created within the page object file for that page only - see storefront-product-details-page.ts as an example of page elements and actions inside a page objects
+- In page object files create ONLY the page elements that are used in the tests, never try to cover all possible elements.
+- Cypress.Commands.add should ONLY be used for cases when the action is not related to any application page: a background operation, a DB query, etc. Do not create Cypress.Commands.add to manipulate the page elements.
+
+### Folder Structure:
+```
+cypress/
+  e2e/                  # container for  test specifications
+    backoffice/         # specifications *.cy.ts that test backoffice features are grouped here, further subdivided into folders by features
+    glue/               # specifications *.cy.ts that test Glue API features are grouped here, further subdivided into folders by features
+    merchant-portal/    # specifications *.cy.ts that test Merchant Portal features are grouped here, further subdivided into folders by features
+    storefront/         #specifications *.cy.ts that test Storefront features are grouped here, further subdivided into folders by features
+  fixtures/             # typed JSON fixtures that store test data
+  support/
+    cy-commands/        # *.ts files with custom commands that cannot be inside a scenario or a page object, e.g. a DB manipulation command
+    glue-endpoints/     # *.ts files each of which contains ulrs, payload, test data references and headers for one specific API endpoint that can be reused inside a test
+    page-objects/       # *.ts files each of which contains a page object - selectors and actions that are limited to a single application page
+    scenarios/          # *.ts files that contain complex reusable scenarios that involve actions with more than one page object, so two or more application pages
+    e2e.ts              # global setup, test libs
+    index.d.ts          # TS augmentation for Cypress
+```
+
+## TypeScript & Config
+
+- For TS config use existing `tsconfig.json`
+- For Cypress config use existing `cypress.config.ts`
+- for E2E config use existing `cypress/support/e2e.ts`
+
+## Selectors & Page Objects
```

Comment:
Could you please check if it's worth adding examples from 'core' rules, like here https://github.com/spryker-dev/prompt-library/blob/main/rules/cypress/cypress-tests.md?plain=1#L97

Reply:
added